packages/next/server/lib/squoosh/avif/avif_node_dec.js

```diff
@@ -0,0 +1,1794 @@
+/* eslint-disable */
+var Module = (function () {
+  return function (Module) {
+    Module = Module || {}
+
+    var Module = typeof Module !== 'undefined' ? Module : {}
+    var readyPromiseResolve, readyPromiseReject
+    Module['ready'] = new Promise(function (resolve, reject) {
+      readyPromiseResolve = resolve
+      readyPromiseReject = reject
+    })
+    var moduleOverrides = {}
+    var key
+    for (key in Module) {
+      if (Module.hasOwnProperty(key)) {
+        moduleOverrides[key] = Module[key]
+      }
+    }
+    var arguments_ = []
+    var thisProgram = './this.program'
+    var quit_ = function (status, toThrow) {
+      throw toThrow
+    }
+    var ENVIRONMENT_IS_WEB = false
+    var ENVIRONMENT_IS_WORKER = false
+    var ENVIRONMENT_IS_NODE = true
+    var scriptDirectory = ''
+    function locateFile(path) {
+      if (Module['locateFile']) {
+        return Module['locateFile'](path, scriptDirectory)
+      }
+      return scriptDirectory + path
+    }
+    var read_, readBinary
+    var nodeFS
+    var nodePath
+    if (ENVIRONMENT_IS_NODE) {
+      if (ENVIRONMENT_IS_WORKER) {
+        scriptDirectory = require('path').dirname(scriptDirectory) + '/'
+      } else {
+        scriptDirectory = __dirname + '/'
+      }
+      read_ = function shell_read(filename, binary) {
+        if (!nodeFS) nodeFS = require('fs')
+        if (!nodePath) nodePath = require('path')
+        filename = nodePath['normalize'](filename)
+        return nodeFS['readFileSync'](filename, binary ? null : 'utf8')
+      }
+      readBinary = function readBinary(filename) {
+        var ret = read_(filename, true)
+        if (!ret.buffer) {
+          ret = new Uint8Array(ret)
+        }
+        assert(ret.buffer)
+        return ret
+      }
+      if (process['argv'].length > 1) {
+        thisProgram = process['argv'][1].replace(/\\/g, '/')
+      }
+      arguments_ = process['argv'].slice(2)
+      quit_ = function (status) {
+        process['exit'](status)
+      }
```

Comment:
Glad that they've removed the `process.on('uncaughtException')` event handlers. 👍